### DISCUSSION TITLE
How run airfow docker-compose with open-telementry?

### DISCUSSION THREAD

**Discussion by C-h-e-r-r-y** [apache/airflow#39835] [Q&A]
Here is a [link](https://github.com/apache/airflow/blob/main/scripts/ci/docker-compose/integration-otel.yml) to `integration-otel.yml` file which can be run by docker. The problem is that `airflow:` section do not contains any image or other settings. it is unclear how to run it. Could somebody please clarify how run airflow open telemetry demo from `integration-otel.yml` file?

Thanks in advance!

**Comment by potiuk** [comment]
It's not a demo. It's airflow development setting for contributors. You run it with `breeze` and it's part of integration tests. https://github.com/apache/airflow/blob/main/contributing-docs/testing/integration_tests.rst  it's only part of the suite and it only works together with other docker-compose files from that same folder.

You migh want to learn how breeze work and run it following https://github.com/apache/airflow/blob/main/contributing-docs/06_development_environments.rst#breeze-development-environment.

That's all there is to it. there is no "user" facing configuration for docker-compose, you can take whatever docker-compose you have and use that one as inspiration, but this is as much of a resource as you can get.
